src/core/public/core_system.ts

```diff
@@ -17,13 +17,14 @@
  * under the License.
  */
 
+import './core.less';
```

Comment:
It's possible to use sass, but it requires changes to the sass infrastructure that I don't really feel justifies what we're trying to accomplish here. I'll switch over to CSS if that's alright. If these styles grow, or we seek to get rid of all style support in the optimizer, the changes I'm imagine will be necessary to the sass infrastructure will be necessary for other styles too so I think it will be more worthwhile to transition to sass then.

Reply:
Yeah, let's not overhaul anything in order to add 3 lines of styles here. Pure css works for me if that's doable.